_posts/2016-10-10-Help!-I'm-becoming-Post-Junior.md

```diff
@@ -111,6 +111,16 @@ A lot of people’s work relies on Open Source code and as you are starting to b
 
 Interacting in these projects exposes you to whole new teams of people with, hopefully, very different perspectives. It will change you, you will change them.
 
+## Moving On
+
+This is a [complicated topic][23], perhaps worth of it’s own post.
+
+When you first start looking for a job in technology, you likely didn’t have too many choices and was pleased to have any offer. By this point, I’m hoping you’re at a point where you understand your value to a company. 
+
+Perhaps it’s worth thinking about what kind of space the company has for you to grow in. Since I started my career, Artsy is the only job I’ve been in for longer than 2 years. For some, I felt like I had outgrown my original role but could not find a space to grow into. 
+
+It’s worth re-evaluating. I do it every year in anticipation of my [annual write-ups][24].
```

Comment:
The one thing I'd suggest, especially for developers in their early career, is to try to stay until they have a concrete accomplishment to hang their hat on. Sometimes, it's not possible, if it's just a bad fit or poor timing. Being able to speak to their specific contributions in tasks in larger project is critical to advancing up the career ladder.

But yeah, much could be said here.